src/main/java/com/google/sps/data/Guide.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.data;
+
+public class Guide {
+    private final int id;
+    private final String title;
+    private final String description;
+    private final String content;
+    private final long timestamp;
+
+    public Guide(int id, String title, String description, String content){
+        this.id = id;
+        this.title = title;
+        this.description = description;
+        this.content = content;
+        this.timestamp = System.currentTimeMillis();
```

Comment:
You'll want a constructor where you can pass this in. That way it can be a value read from the database.

Reply:
Addressed in next commit.